README.md

```diff
@@ -0,0 +1,129 @@
+# IPNS
+
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://protocol.ai)
+[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+[![standard-readme](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+[![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/feross/standard)
+
+> ipns record definitions
+
+This module contains all the necessary code for creating, understanding and validating IPNS records.
+
+## Lead Maintainer
+
+[Vasco Santos](https://github.com/vasco-santos).
+
+## Table of Contents
+
+- [Install](#install)
+- [Usage](#usage)
+  - [Create Record](#create-record)
+  - [Validate Record](#validate-record)
+  - [Embed public key to record](#embed-public-key-to-record)
+  - [Extract public key from record](#extract-public-key-from-record)
+  - [Datastore key](#datastore-key)
+- [API](#api)
+- [Contribute](#contribute)
+- [License](#license)
+
+### Install
+
+> npm install ipns
+
+## Usage
+
+#### Create record
+
+```js
+const ipns = require('ipns')
+
+ipns.create(privateKey, value, seqNumber, eol, (err, entryData) => {
+  // your code goes here
+});
+```
+
+#### Validate record
+
+```js
+const ipns = require('ipns')
+
+ipns.validate(publicKey, ipnsEntry, (err) => {
+  // your code goes here
+  // if no error, the record is valid
+});
+```
+
+#### Embed public key to record
+
+> Not available yet
+
+#### Extract public key from record
+
+> Not available yet
+
+#### Datastore key
+
+```js
+const ipns = require('ipns')
+
+ipns.getDatastoreKey(peerId);
+```
+
+Returns a key to be used for storing the ipns entry in the datastore according to the specs, that is:
```

Comment:
Please link to specs you're referring to or reword...

Reply:
I will have to reword by now! I still didn't create the PR with specs.